app/autoload.php

```diff
@@ -2,6 +2,8 @@
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
 
+error_reporting(error_reporting() & ~E_USER_DEPRECATED);
```

Comment:
This is a stupid copy paste from Symfony standard edition: https://github.com/symfony/symfony-standard/blob/2.8/app/autoload.php#L6

Do you still want to remove this?